### PR TITLE
Fix #1019: Add 'Find in page' to search panel.

### LINKED ISSUE #1019: Bring back `Find in page` via url bar
We had this feature in Brave 1.6 and removed it by accident while moving onto 1.7 codebase.

### PR DESCRIPTION
**Pull request by iccub**
## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-ios/issues) for my issue if one did not already exist.
- [x] My patch or PR title has a standard commit message that looks like `Fix #123: This fixes the shattered coffee cup!` (or `No Bug: <message>` if no relevant ticket)
- [ ] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`
- [ ] New files have MPL-2.0 license header.

## Reviewer Checklist:

- [x] PR is linked to an issue via [Zenhub](https://www.zenhub.com/extension).
- [x] Issues are assigned to at least one epic.
- [x] Issues include necessary QA labels:
  - [x] `QA/(Yes|No)`
  - [x] `release-notes/(include|exclude)`
  - [x] `bug` / `enhancement`
- [ ] Necessary security reviews have taken place.
- [ ] Adequate test coverage exists to prevent regressions.
- [ ] Adequate test plan exists for QA to validate (if applicable)

### CODE REVIEW

**Comment by iccub** [comment]
reference screenshot

Zrzut ekranu 2019-07-8 o 15 22 49

File: BraveShared/BraveStrings.swift [hunk 1]
```diff
@@ -117,6 +118,10 @@ extension Strings {
...
     public static let TabToolbarReaderViewButtonAccessibilityLabel = NSLocalizedString("TabToolbarReaderViewButtonAccessibilityLabel", tableName: "BraveShared", bundle: Bundle.braveShared, value: "Reader View", comment: "Accessibility label for the Reader View button")
     public static let TabToolbarReaderViewButtonTitle = NSLocalizedString("TabToolbarReaderViewButtonTitle", tableName: "BraveShared", bundle: Bundle.braveShared, value: "Add to Reading List", comment: "Accessibility label for action adding current page to reading list.")
+    public static let FindOnPageSectionHeader = NSLocalizedString("FindOnPageSectionHeader", tableName: "BraveShared", bundle: Bundle.braveShared, value: "On this page", comment: "Section header for find in page option")
+    public static let SearchHistorySectionHeader = NSLocalizedString("FindOnPageSectionHeader", tableName: "BraveShared", bundle: Bundle.braveShared, value: "History and bookmarks", comment: "Section header for history and bookmarks option")
+
```
Comment (kylehickinson) [inline BraveShared/BraveStrings.swift]: Whats this whitespace for? lol

File: Client/Frontend/Browser/BrowserViewController.swift [hunk 2]
```diff
@@ -1948,6 +1948,23 @@ extension BrowserViewController: SearchViewControllerDelegate {
...
+    func searchViewController(_ searchViewController: SearchViewController, shouldFindInPage query: String) {
+        topToolbar.leaveOverlayMode()
+        updateFindInPageVisibility(visible: true)
+        findInPageBar?.text = query
+        
+    }
+    
+    func searchViewControllerAllowFindInPage() -> Bool {
+        // Hides find in page for new tabs.
+        if let st = tabManager.selectedTab, let wv = st.webView {
```
Comment (kylehickinson) [inline Client/Frontend/Browser/BrowserViewController.swift]: Cant you reduce this to one line:

`return tabManager.selectedTab?.webView?.url?.isAboutHomeURL == false`

**Review by kylehickinson — CHANGES_REQUESTED** [review]
Just minor nits, and a bunch of whitespace to cleanup :)

File: BraveShared/BraveStrings.swift (hunk 1 above)
Comment (iccub) [reply BraveShared/BraveStrings.swift]: it's for style points

File: Client/Frontend/Browser/BrowserViewController.swift (hunk 2 above)
Comment (iccub) [reply Client/Frontend/Browser/BrowserViewController.swift]: Copy pasted from 1.6 🤡

**Review by kylehickinson — APPROVED** [review]

**Pull request merged**